### PR TITLE
Discord api

### PR DESCRIPTION
**Pull request by Avev**
worked on #3 
Created an event driven discord bot that tracks member and project activity.

### CODE REVIEW

File: airflow/scripts/discord_scripts/discord_bot.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import discord
+from dotenv import load_dotenv
+import os
+
+intents = discord.Intents.default()
+intents.members = True
+
+load_dotenv()
+
+BOT_TOKEN = os.getenv('BOT_TOKEN')
```
Comment (eyalFischel) [inline airflow/scripts/discord_scripts/discord_bot.py:12]: Where is your .env file ? are you hiding it with .gitignore? if so , where is the .gitignore update on the root?

File: airflow/scripts/discord_scripts/discord_bot.py [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import asyncio
+from datetime import datetime, timedelta
```
Comment (eyalFischel) [inline airflow/scripts/discord_scripts/discord_bot.py:2]: Order it by import types and alphabetic:
<local python imports>

<third party>

<your imports>

example:
import asyncio
from datetime import datetime, timedelta
import os

import discord
from dotenv import load_dotenv

File: airflow/scripts/discord_scripts/discord_bot.py [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+import discord
+from dotenv import load_dotenv
```
Comment (eyalFischel) [inline airflow/scripts/discord_scripts/discord_bot.py:4]: Where is your requirement file for your script?

File: airflow/scripts/discord_scripts/discord_bot.py [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+    
+    update_member_activity(message)
+    if message.channel.category and is_projects_category(message.channel.category):
+        update_project_activity(message)
+
+    
+@client.event
+async def on_member_join(member):
+    if member.guild.id == GUILD_ID:
+        global join_count
```
Comment (eyalFischel) [inline airflow/scripts/discord_scripts/discord_bot.py:44]: avoid global if possible

File: airflow/scripts/discord_scripts/discord_bot.py [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        await asyncio.sleep(wait_time)
+
+        global join_count
+        join_count = 0
+
+        print("Weekly counter reset " + datetime.now())
+        await asyncio.sleep(7*24*60*60)
+
+
+def is_projects_category(category):
```
Comment (eyalFischel) [inline airflow/scripts/discord_scripts/discord_bot.py:68]: category is a string , list? add to it the type. 
Same about the functions below , typing and what they return.

**Review by eyalFischel — COMMENTED** [review]
Overall the code is short and great. Try to move 68 - 97 to utils.py

File: airflow/scripts/discord_scripts/discord_bot.py (hunk 1 above)
Comment (Avev) [reply airflow/scripts/discord_scripts/discord_bot.py]: My .env is in the discord_scripts file, I added a .gitignore on the root

**Pull request merged**